perlmod/Fink/Services.pm

```diff
@@ -594,6 +599,32 @@ EOSCRIPT
 		@wrap = map "$_=$ENV{$_}", sort keys %ENV;
 		push @wrap, "__CFPREFERENCES_AVOID_DAEMON=1";
 		unshift @wrap, 'env' if @wrap;
+		my $runtime_request = Fink::Config::get_option("build_in_sandbox");
```

Comment:
If you have a suggested patch to achieve the desired behavior, I will be very happy to do the complete test cycle here later in the week. Again, the behavior we are targeting is...

1) fink defaults to sandboxing unless %p/etc/fink.sb is an empty file or is missing entirely.,
2) The 'NoSandbox: true' info file parameter allows the default sandboxing to be disabled but can be overridened again with the fink command line option '--build-in-sandbox'.
3) The '-no-build-in-sandbox' fink command line option has precedence over the 'NoSandbox: false' info file parameter.

The current code meets those design targets.

ps I am at my coding limits in perl and need a fresh set of eye-balls on the problems to find a different coding path.